packages/clay-dropdown/src/ClayDropdownBase.js

```diff
@@ -85,13 +230,28 @@ ClayDropdownBase.STATE = {
 	caption: Config.string(),
 
 	/**
+	 * A map from `Align` position constants to the CSS class that should be
+	 * added to the dropdown when it's aligned in that position.
+	 * @instance
+	 * @memberof ClayDropdownBase
+	 * @type {!Object}
+	 * @default valueClassMapFn_
+	 */
+	classMap_: Config.validator(object)
```

Comment:
classMap was used in metal-dropdown soy template. Since we're not using this param in our templates we can get rid of this.